docs/Onboarding/Prerequisites_Guide.md

```diff
@@ -0,0 +1,90 @@
+# (WIP) OpenShift CI Scenario Prerequisites Guide<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Overview](#overview)
+- [Prerequisites at a Glance](#prerequisites-at-a-glance)
+- [Resources](#resources)
+- [Upstream Considerations](#upstream-considerations)
+  - [1. Public GitHub repositories](#1-public-github-repositories)
+  - [2. Private GitHub repositories](#2-private-github-repositories)
+- [GitHub Org Requirements](#github-org-requirements)
+- [GitHub User Readiness](#github-user-readiness)
+- [Containerized Product Deployment](#containerized-product-deployment)
+- [Containerized Tests](#containerized-tests)
+- [Resolve Internal Dependencies](#resolve-internal-dependencies)
+- [Valid Test Output](#valid-test-output)
+- [Documented Tests](#documented-tests)
+
+## Overview
+This doc is meant to be used by Product QE teams to prepare themselves and their testing repositories to run within OpenShift CI.
+
+You will be working through and updating the following information in the prerequisite Jira ticket that you will be assigned when starting this prerequisite process.
+
+>Please reach on the Slack channel [#forum-qe-cspi-ocp-ci](https://coreos.slack.com/archives/C047Y0DPEJU) to notify us that you want to start the prerequisite process so that we can get you started with a prerequisite ticket.
```

Comment:
```suggestion
>Please reach out to us on the Slack channel [#forum-qe-cspi-ocp-ci](https://coreos.slack.com/archives/C047Y0DPEJU) to let us know that you want to start the prerequisite process so that we can get you started with a prerequisite ticket.
```